code/obj/item/misc_wizard_stuff.dm

```diff
@@ -165,8 +165,9 @@
 	desc = "A dark staff infused with eldritch power. Trying to steal this is probably a bad idea."
 	icon_state = "staffcthulhu"
 	item_state = "staffcthulhu"
-	force = 14
+	force = 0
 	hitsound = 'sound/effects/ghost2.ogg'
+	var/magic_damage = 14
```

Comment:
I only made this a var incase someone wants to ~~commit crimes~~ varedit for whatever reason.

Reply:
Note that this makes it unable to say, bash a window. Intentional?

Reply:
It's funny that I never seriously considered someone might actually use this for anything other than bashing someone's head in. I think it's a bit silly that the staff doesn't do strictly "magical" damage and can actually be used to bash other things down, but that's outside the scope of this PR and a discussion that I'd rather not start here. 

Reverted to a previous revision.